### ISSUE TITLE
Sale Tax

### ISSUE THREAD

**Issue by suryak-dev** [thephpleague/omnipay-authorizenet#138]
Hi there, can you please give me an example of how can I charge a tax amount?

**Comment by judgej** [comment]
I don't believe this driver supports VAT, unfortunately. Most drivers support the cart object that allows a breakdown of payments to be included (and even that does not support VAT out of the box, though some drivers extend it to add VAT support.

The APIs in this driver (AIM, CIM, SIM, DPM) are all deprecated by Authorize.Net anyway, so are unlikely to get much attention. There is now a single REST API that supports multiple ways to integrate.
